packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2829,20 +2829,20 @@ export class StripeHelper extends StripeHelperBase {
     } = productNewMetadata;
     const planConfig = await this.maybeGetPlanConfig(planIdNew);
 
-    const productPaymentCycleNew = this.stripePlanToPaymentCycle(planNew);
+    const productPaymentCycleNew = await this.stripePlanToPaymentCycle(planNew);
```

Comment:
Once the `async` is removed above, this await is no longer necessary and can also be removed.